.vscode/tasks.json

```diff
@@ -0,0 +1,25 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
+    {
+        "label": "build",
+        "command": "./build.cmd",
+        "type": "shell",
+        "args": [
+        ],
+        "problemMatcher": "$msCompile",
+        "group": "build"
+    },
+    {
+        "label": "build skip analyzers",
+        "command": "./build.cmd -skipAnalyzers",
```

Comment:
❔ Why is `-skipAnalyzers` part of `command` instead of `args`?

Reply:
Cause I'm still a noob with Visual Studio code.

Reply:
Also, since it's a `"shell"`-style command, it's just 6 of one, half a dozen of the other.